app/controllers/api/comments_contrroler.rb

```diff
@@ -0,0 +1,27 @@
+class Api::CommentsController < ApplicationController
+  def index
+    @comments = Post.find(params[:post_id]).comments
+    redirect_to do |format|
+      format.json { render json: @comments }
+    end
+  end
+
+  def create
+    @comment = Comment.new(comment_params)
```

Comment:
- You have done a good job so far, but please note that this endpoint should create a comment for a particular post, so it is necessary to specify the post for which the comment is created and also the user creating the comment i.e establishing the `belongs-to` relationship. Please fix this 🙏